README.md

```diff
@@ -64,9 +36,5 @@ setwd("/path/to/analyticsapi-engines-r-sdk")
 Sys.setenv("ANALYTICS_API_URL" = "https://api.factset.com")
 Sys.setenv("ANALYTICS_API_USERNAME_SERIAL" = "<username-serial>")
 Sys.setenv("ANALYTICS_API_PASSWORD" = "<apikey>")
-```
-
-## Contributing
-
-* Files in [auto-generated-sdk](auto-generated-sdk) directory are auto-generated and should not be manually edited here. Refer [Analytics API Engines SDK Generator](https://github.com/factset/analyticsapi-engines-sdk-generator) for instructions on how to modify these files.
-* Projects [examples](examples) and [tests](tests) are open to enhancements and bug fixes. Please create a pull requests with the proposed changes.
+Sys.setenv("CLIENT_PROXY_URL" = "<proxyurl>")PROXY_URL
```

Comment:
Very minor: Is there a typo mistake in this line?

Reply:
it is a type mistake cleaned up